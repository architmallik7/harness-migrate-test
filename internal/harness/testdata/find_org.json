{
    "status": "SUCCESS",
    "data": {
        "organization": {
            "identifier": "default",
            "name": "default",
            "description": "Default Organization",
            "tags": {}
        },
        "createdAt": 1623712669478,
        "lastModifiedAt": 1623712669478,
        "harnessManaged": true
    },
    "metaData": null,
    "correlationId": "0d612bf3-2f3a-40af-87ab-61c3f1634d2c"
}