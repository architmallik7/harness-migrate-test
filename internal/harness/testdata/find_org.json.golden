{
  "identifier": "default",
  "name": "default",
  "description": "Default Organization"
}